### PR TITLE
App overhaul

### PR DESCRIPTION
**Pull request by robertkirsz**
In this PR:
- Up-to-date dependencies
- Authors can add GitHub and Codepen links
- New serif body font, looks quite nice I think
- Better mobile UI (no more horizontal scrolling)
- Bigger content width on desktop
- Misc layout and typography improvements

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     'arrow-spacing': ['warn', { before: true, after: true }],
     indent: 'off',
     'comma-dangle': 'off',
+    'space-before-function-paren': 'off',
```
Comment (robertkirsz) [inline .eslintrc.js]: This clashes with Prettier

File: src/components/GlobalStyle.js [hunk 2]
```diff
@@ -2,8 +2,14 @@ import { createGlobalStyle } from 'styled-components'
...
   body {
-    font: 16px/1.5 Montserrat, sans-serif;
+    font: 14px/1.5 Merriweather, Georgia, serif;
     margin: 0;
+    box-sizing: border-box;
+    color: #333;
+  }
+
+  *, *::after, *::before {
+    box-sizing: inherit;
```
Comment (robertkirsz) [inline src/components/GlobalStyle.js]: Arguably better than a default `box-sizing`: https://css-tricks.com/box-sizing/#article-header-id-6

File: src/components/Author.js [hunk 3]
```diff
@@ -1,28 +1,60 @@
...
+          <a href={`http://codepen.io/${author.codepen}`} target="_blank">
+            <img src={codepenLogo} alt="Codepen logo" width="24" />
+          </a>
+        )}
+
+        {author.github && (
+          <a href={`https://github.com/${author.github}`} target="_blank">
+            <img src={githubLogo} alt="GitHub logo" width="24" />
+          </a>
+        )}
```
Comment (robertkirsz) [inline src/components/Author.js]: Maybe we could loop/map through these somehow? 🤔

Comment (kzadurska) [reply src/components/Author.js]: I have to brush up on my graphql... but assuming we can store it like that 
```
author.links = {
     twitter: <twitter-link>,
     codepen: <codepen-link>
}
```
We could something of
```
Object.keys(author.links).map(link => (
       <a href={author.links[link]}>{link}</a>
))
```

Comment (robertkirsz) [reply src/components/Author.js]: Perfect 👌 ❤️

**Pull request merged**